internal/metrics/remote_write.go

```diff
@@ -138,7 +142,10 @@ func (r *RemoteWrite) Update(config models.DeviceConfigurationMessage) error {
 
 	if featureDisabled {
 		r.client = nil
+		r.currentServerCaFile = ""
+		r.removeServerCaFiles()
 	} else {
+		// parse and validate new config
```

Comment:
The update function becomes too long, please remove that "else" scope to another function

Reply:
done